packages/legend-extension-dsl-data-space/src/graph/metamodel/pure/model/packageableElements/dataSpace/DSL_DataSpace_DataSpace.ts

```diff
@@ -183,6 +184,14 @@ export class DataSpace extends PackageableElement implements Hashable {
   diagrams?: DataSpaceDiagram[] | undefined;
   supportInfo?: DataSpaceSupportInfo | undefined;
 
+  constructor(name: string) {
```

Comment:
as mentioned DO NOT add observables directly on the models. 
This was done to increase performance when editing is not needed. these should follow other elements in adding them as observable helpers.

Reply:
fixed